spec/rails_helper.rb

```diff
@@ -4,6 +4,7 @@
 abort("DATABASE_URL environment variable is set") if ENV["DATABASE_URL"]
 
 require "rspec/rails"
+require "spec_helper"
```

Comment:
I would move this line to be the first line in this file

Reply:
Done in sha: e703b14